Leaderboard/locals.js

```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash');
+
+const locals = {
+  // App configuration
+  appName: 'Fight or Flight',
+  primaryColor: 'indigo',
```

Comment:
This should probably be moved to the config folder (default.yml), but these are all variables that are available to the templating engine.